core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -813,6 +825,18 @@ class EMConfig {
     @Cfg("Enable extracting SQL execution info")
     var extractSqlExecutionInfo = true
 
+    @Experimental
+    @Cfg("Tracking of MongoDB commands to improve test generation")
+    var heuristicsForMongo = false
+
+    @Experimental
+    @Cfg("Enable extracting MongoDB execution info")
+    var extractMongoExecutionInfo = true
```

Comment:
if it is experimental, i guess it should be `false` by default

Reply:
ok